src/test/suite/extension.test.ts

```diff
@@ -8,4 +12,15 @@ suite('Extension Test Suite', () => {
 		assert.ok(vscode.extensions.getExtension(extensionId));
 		done();
 	});
+
+	test('by default, didact setting to open on startup should be false', async function (done) {
+		const openAtStartup : boolean = getOpenAtStartupSetting();
+		console.log(`openAtStartup = ${openAtStartup}`);
+		if (openAtStartup === true) {
```

Comment:
I think the `=== true` is not really required here. A simple `if (openAtStartup)` would do it too. But thats really minor.

Reply:
Unfortunately it seems to be required to work correctly on Travis. So I've left this in